### PR TITLE
add change log

### PR DESCRIPTION
**Pull request by mrow4a**

### CODE REVIEW

**Comment by PVince81** [comment]
no entries as this is the initial version ?

**Comment by mrow4a** [comment]
I mean, for now there are nearly none, maybe the icon and LDAP problem. I did not manage to make the build/gen_changelog.sh script run on this app

**Comment by PVince81** [comment]
@mrow4a try this: `vim .git/config` and append ".git" at the end of the repo URL.

**Comment by PVince81** [comment]
since there wasn't any previous releases, there aren't any changes to be documented here.

Please add a "Initial release" entry and remove the empty sections.

**Comment by pmaier1** [comment]
Please adapt and merge this until Thursday for that we can release with 10.0.3. Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
```
Comment (PVince81) [inline CHANGELOG.md]: remove this empty section

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## 0.1.1
```
Comment (PVince81) [inline CHANGELOG.md]: I assume we agreed that 0.1.1 will be the released version and no one will bump it ?

**Comment by PVince81** [comment]
Okay I see that this is the last item in https://github.com/owncloud/diagnostics/issues/12, so just remove "Unreleased" and make "0.1.1" clickable to see the commits until v0.1.1.

**Comment by mrow4a** [comment]
Hmm, but according to changelog documentation this section can be empty and is just a shortcut to commits which are unreleased @PVince81

**Pull request merged**